### PR TITLE
Dev/130 [ExamQuestion] Question order jumping number

### PR DESCRIPTION
**Pull request by Wally5077**
**before fix:**

---

**after fix:**

**before adding:**

**after adding:**

### CODE REVIEW

File: src/app/exam/questions/ExamQuestions.js [hunk 1]
```diff
@@ -101,58 +102,52 @@ const ExamQuestions = () => {
...
-                                    {rejudgingProblemId === question.problemId ?
-                                        <span className="tag"
-                                              style={{
-                                                  backgroundColor: "#FFBB00",
-                                                  color: "white",
-                                                  width: "75px"
-                                              }}>
+                        data: (question) => [
+                            <p>{toCharacterIndex(questions.findIndex(_question => _question.questionOrder === question.questionOrder))}</p>,
+                            <FakeLink>{`${question.problemId} ${question.problemTitle}`}</FakeLink>,
```
Comment (Chaoyuuu) [inline src/app/exam/questions/ExamQuestions.js:117]: Can you change this to the real link?
When clicking the questions, it should redirect to the related problem editor.

File: src/app/exam/questions/ExamQuestions.js [hunk 2]
```diff
@@ -101,58 +102,52 @@ const ExamQuestions = () => {
...
+                            <p className="text-center">{question.maxScore}</p>,
+                            <p className="text-center">{question.quota}</p>,
+                            <div>
+                                {rejudgingProblemId === question.problemId ?
+                                    <span className="tag"
+                                          style={{
+                                              backgroundColor: "#FFBB00",
+                                              color: "white",
+                                              width: "75px"
+                                          }}>
```
Comment (Chaoyuuu) [inline src/app/exam/questions/ExamQuestions.js:127]: Can you move this style into `.scss`?  
And please update the backgroundColor to `#FFC700`.

**Review by Chaoyuuu — CHANGES_REQUESTED** [review]
Review Done!

**Comment by Chaoyuuu** [comment]
Hey, here are some comments. 
- I think this is a duplicate code. We have the same style in the `examQuestions.scss`. 
image

- Does `color:#FFF` really work? Please fix it. Thanks.

File: src/app/exam/questions/ExamQuestions.js [hunk 3]
```diff
@@ -101,58 +102,52 @@ const ExamQuestions = () => {
...
+                        ]
                     }}
                     showFilterSearchBar={false}/>
                 <div className="add-question-btn"
                      onClick={() => setShowAddQuestionModal(true)}
-                     style={{
-                         alignSelf: "flex-end",
-                         position: "relative"
-                     }}>
+                     style={{alignSelf: "flex-end", position: "relative"}}>
```
Comment (Johnny850807) [inline src/app/exam/questions/ExamQuestions.js:150]: write it into the scss file

**Review by Johnny850807 — CHANGES_REQUESTED** [review]
Review done

File: src/app/exam/questions/ExamQuestions.scss [hunk 4]
```diff
@@ -11,14 +11,33 @@
...
     border-radius: 50px;
     display: inline;
     margin: 1rem 0;
     padding: 5px 20px;
     cursor: pointer;
+    align-self: flex-end;
+    position: relative;
 
     span {
       color: #FFF;
```
Comment (Chaoyuuu) [inline src/app/exam/questions/ExamQuestions.scss:43]: please remove it if we don't need it.

**Review by Chaoyuuu — CHANGES_REQUESTED** [review]
Review done

**Review by Johnny850807 — APPROVED** [review]
ＬＧＴＭ

**Review by Chaoyuuu — APPROVED** [review]
LGTM

**Pull request merged**